.github/workflows/rw-entry-testing-baseset.yml

```diff
@@ -45,4 +45,6 @@ jobs:
     needs:
     - build
 
+    if: always() && github.event_name == 'pull_request'
```

Comment:
This would now only run on PR, but currently it also runs after merge. Is that change intended?

Reply:
Okay, so mostly the `rw-annotation-check.yml` is outdated, and that `if` condition should be moved up one. So that `if` condition doesn't actually do what we hope/think it does :) At least, the `always()` part there is pointless, if I get it right.

Maybe good to just remove the `if` statement in that workflow. Makes this PR more clear in its intentions, and also forces us to cleanup some other repos that wrongly don't have the `if` statement before using the `rw-annotation-check.yml`. If that makes sense :P (please check if it makes sense)

Reply:
Scratch that: my suggestion, for this PR: replace the `rw-annotation-check.yml` below with the actual action directly (so the `annotation-check`). And leave the `rw-annotation-check.yml` alone for now.

There are three users of that `rw-`. I think we should fix them to work how they are intended (they miss the `if` statement), and switch to the `annotation-check` directly too, much like the other repos like OpenTTD, catcodec, ...

That way, all repos are the same again, with the least amount of effort :P My suggestion, feel free to look around yourself what you think is best. But this did get kinda messy :D